src/app/lib/access/mocking/mocking_repository.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+abstract interface class MockingRepository {
+  factory MockingRepository() = _MockingRepository;
+
+  /// Gets whether the mocking is enabled.
+  Future<bool> checkMockingEnabled();
```

Comment:
`checkMockingEnabled` doesn't seem like a conventional name for a method that gets a value.